tests/compatibility_issues.py

```diff
@@ -12,269 +11,182 @@
+    """and the duplicate will be ignored""",
+    "duplicate_in_other_calendar_with_same_uid_breaks": """Fetch an event from one calendar, save it to another ... """
+    """and get some error from the server""",
+    "event_by_url_is_broken": """A GET towards a valid calendar object resource URL will yield 404 (wtf?)""",
+    "uid_required": """Server will not accept calendar object resources without an UID""",
+    "no_sync_token": """RFC6578 is not supported, things will break if we try to do a sync-token report""",
+    "time_based_sync_tokens": """The sync token is typically a time stamp, and we need to sleep a """
+    """second in the test code to get things right""",
+    "fragile_sync_tokens": """Every now and then (or perhaps always), more content than expected """
+    """will be returned on a simple sync request.  Possibly a race condition """
+    """if the token is timstamp-based?""",
+    "sync_breaks_on_delete": """I have observed a calendar server (sabre-based) that returned """
+    """418 I'm a teapot """
+    """when requesting updates on a calendar after some calendar resource """
+    """object was deleted""",
+    "propfind_allprop_failure": """The propfind test fails ... """
+    """it asserts DAV:allprop response contains the text 'resourcetype', """
+    """possibly this assert is wrong""",
+    "no_todo": """Support for VTODO (tasks) apparently missing""",
+    "no_todo_datesearch": """Date search on todo items fails""",
+    "vtodo_datesearch_nodtstart_task_is_skipped": """date searches for todo-items will not find tasks without a dtstart""",
+    "vtodo_datesearch_nodtstart_task_is_skipped_in_closed_date_range": """only open-ended date searches for todo-items will find tasks without a dtstart""",
+    "vtodo_datesearch_notime_task_is_skipped": """date searches for todo-items will (only) find tasks that has either """
+    """a dtstart or due set""",
+    "vtodo_no_due_infinite_duration": """date search will find todo-items without due if dtstart is """
+    """before the date search interval.  I didn't find anything explicit """
+    """in The RFC on this (), but an event should be considered to have 0 """
+    """duration if no dtend is set, and most server implementations seems to """
+    """treat VTODOs the same""",
+    "no_todo_on_standard_calendar": """Tasklists can be created, but a normal calendar does not support tasks""",
+    "unique_calendar_ids": """For every test, generate a new and unique calendar id""",
+    "sticky_events": """Events should be deleted before the calendar is deleted, """
+    """and/or deleting a calendar may not have immediate effect""",
+    "object_by_uid_is_broken": """calendar.object_by_uid(uid) does not work""",
+    "no_mkcalendar": """mkcalendar is not supported""",
+    "no_overwrite": """events cannot be edited""",
 }
 
 xandikos = [
     ## TEMP TEMP TEMP - TODO - should be investigated
     ## (perhaps my xandikos version is too old?)
-    "no_expand", "no_recurring",
```

Comment:
Another comment context that gets broken by "correct" formatting.  The comment above applies to both no_expand and no_recurring, which are intentionally placed on the same line to make it easy to remove them at a later stage.